dashboard/app/models/foorm/form.rb

```diff
@@ -48,7 +52,7 @@ def self.setup
     end
 
     transaction do
-      reset_db
+      Foorm::Form.delete_all
```

Comment:
I changed to delete_all here because I noticed on local the changes did not get rolled back if there was a failure on import and we used reset_db. The ids still got reset on local. We can keep an eye to see if the ids still get reset in other environments, but it's not a big deal if the ids change since we do not rely on them at all.

Reply:
Hmm, any idea why?  `reset_db` [appears to call](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/seeded.rb#L8) `delete_all`.

Reply:
`reset_autoincrement` seems to mess with the ability to do a rollback--when I commented it out in `reset_db` the rollback was successful, otherwise the database was empty if the import failed. I posted about it on slack [here](https://codedotorg.slack.com/archives/C0T10HG6N/p1599172158014800)

Reply:
It could be an issue with local but I didn't want to risk it since import is more likely to fail now.

Reply:
Interesting, thanks for the details.